File: Javascript/POO/classes.js
class Data {
    constructor(type, description){
        this.type = type;
        this.description = description;
    }
    read(){
        console.log(`Type: ${this.type} \nDescription: ${this.description}`);
    }
}

const booksData = new Data('List', 'A list of booking data from a Hotel');
booksData.read();

File: Classes/classes.py
class Data:
    def __init__(self, type, description):
        self.type = type
        self.description = description
    def read(self):
        print('Type:', self.type,'\nDescription:', self.description)
booksData = Data('List', 'A list of booking data from a Hotel')
booksData.read()